Makefile

```diff
@@ -10,9 +10,14 @@ prepare-gcuk:
 
 flash-gcuk: prepare-gcuk perform-flash
 
-prepare-gcus:
+prepare-gcus22:
 	go run cmd/main.go -conf=gcus22
 
+flash-gcus22: prepare-gcus22 perform-flash
+
+prepare-gcus:
```

Comment:
For maintainability, we should refactor to be able to the pass the year at some point :)